tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -28,7 +27,7 @@ import I18nProvider from 'core/i18n/Provider';
 import Rating from 'ui/components/Rating';
 import { fakeAddon, fakeReview } from 'tests/unit/amo/helpers';
 import {
-  apiResponsePage, getFakeI18nInst, unexpectedSuccess,
+  apiResponsePage, createFetchAddonResult, getFakeI18nInst, unexpectedSuccess,
```

Comment:
I prefer each of these methods on their own line but I get this was already this way before. I feel like we've preferred:

```js
import {
  bar,
  foo,
  etc,
} from 'whatever';
```

going forward but we don't have an eslint rule so I won't complain if this lands 😉